bpf/src/cgnet.c

```diff
@@ -19,7 +19,7 @@
 #include <uapi/linux/ip.h>
 #include "bpf_helpers.h"
 
-struct bpf_map_def SEC("maps/count") count_map = {
+struct bpf_map_def SEC("maps/cgnet") count_map = {
```

Comment:
Could you use the same name for the section and the variable name?
something like:
```
struct bpf_map_def SEC("maps/count_map") count_map = {
```